BlazarUI/app/scripts/components/repo-build/InterProjectAlert.jsx

```diff
@@ -0,0 +1,163 @@
+import React, {Component, PropTypes} from 'react';
+import {isEmpty, bindAll} from 'underscore';
+import {Link} from 'react-router';
+import Alert from 'react-bootstrap/lib/Alert';
+import {contains} from 'underscore';
+import classNames from 'classnames';
+
+const initialState = {
+  expanded: false
+};
+
+class InterProjectAlert extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = initialState;
+    bindAll(this, 'onClickAlert');
+  }
+
+  getClassNames() {
+    return classNames([
+      'inter-project-alert', {
+        'expanded': this.state.expanded
+      }
+    ]);
+  }
+
+  getAlertClass() {
+    const {state} = this.props.upAndDownstreamModules;
+
+    if (contains(['QUEUED', 'IN_PROGRESS'], state)) {
```

Comment:
for JS, sometimes people like doing this kind of logic with a map instead of a series of if / case statements